frontend/src/components/wallet/useWalletProvider.ts

```diff
@@ -9,9 +10,15 @@ import { useEffect, useMemo, useReducer, useState } from 'react';
 import Web3Modal, { getInjectedProvider, IProviderInfo } from 'web3modal';
 import { walletReducer } from './WalletReducer';
 
+const ethProvider = require('eth-provider');
```

Comment:
Use imports instead of require

Reply:
there are no types provided for this package yet and I'm not very familiar with that part as I'm a bit new with TS type declarations, a bit of help would be greatly appreciated thanks :)

Reply:
ok so I added a module type declaration file for 'eth-provider' but I'm really not sure if this is the right way to do it. Even tho it does seem to work even if it doesn't actually provide any types.

Reply:
at frontend/src/types/eth-provider.d.ts:
```ts
declare module 'eth-provider';
```